### ISSUE TITLE
Special Characters Corrupt Formatting

### ISSUE THREAD

**Issue by NexRX** [ony3000/prettier-plugin-classnames#50]
## Dependency information
Here is my config
```js
export default {
  plugins: [
    "prettier-plugin-astro", // ^0.13.0
    "prettier-plugin-tailwindcss", // ^0.5.11
    "prettier-plugin-classnames", // ^0.6.4
    "prettier-plugin-merge", // ^0.6.0
  ],
  trailingComma: "none",
  semi: false,
  printWidth: 100,
  tailwindFunctions: ["tw"],
  customAttributes: ["class", "className", "style", "classList"],
  overrides: [
    {
      files: "*.astro",
      options: {
        parser: "astro",
      },
    },
  ],
};
```

## Steps to reproduce
Format a file with this character '•' (there are others) 
`page.astro`
```astro
---
import SettingsLayout from "@layouts/settings.astro";
import SecurityForm from "./_security";
---

<SettingsLayout
  title="Settings • Details"
  class="flex max-w-[1100px] flex-col gap-4 overflow-x-hidden text-left"
>
  <SecurityForm client:only />
</SettingsLayout>
```

## The current behavior
Becomes this
```astro
---
import SettingsLayout from "@layouts/settings.astro";
import SecurityForm from "./_security";
---

<SettingsLayout
  title="Settings • Details"
  class="f"μα"
    " δξξπξκκγαεζμξζμγηκκοιζιζξηιθοημιγνθλ______________________
  γ
  <SecurityForm
  client:only
/>
```
The curruption depends on the character. For example, the string `δξξπξκκγαεζμξζμγηκκοιζιζξηιθοημιγνθλ______________________` is always observed when the character `•` is present. 

## The expected behavior
Nothing should happen in this case as the className is not long enough.

**Comment by ony3000** [comment]
Hello! Thanks for reporting the issue!

Until this issue is fixed, you can work around it using one of the following methods:

1. Use unicode escape sequences instead of special characters.
(Note that it is not written as an expression.)

   ```astro
   ---
   import SettingsLayout from "@layouts/settings.astro";
   import SecurityForm from "./_security";
   ---

   <SettingsLayout
     title="Settings \u2022 Details"
     class="flex max-w-[1100px] flex-col gap-4 overflow-x-hidden text-left"
   >
     <SecurityForm client:only />
   </SettingsLayout>
   ```

2. Write lines containing special characters further down than lines containing the class name.

   ```astro
   ---
   import SettingsLayout from "@layouts/settings.astro";
   import SecurityForm from "./_security";
   ---

   <SettingsLayout
     class="flex max-w-[1100px] flex-col gap-4 overflow-x-hidden text-left"
     title="Settings • Details"
   >
     <SecurityForm client:only />
   </SettingsLayout>
   ```

**Comment by ony3000** [comment]
@NexRX Fixed in `v0.6.5`, check it out!
